tests/test_uharfbuzz.py

```diff
@@ -283,6 +300,29 @@ def test_message_func_crash(self, blankfont):
         hb.shape(blankfont, buf)
 
 
+    def test_draw_funcs(self, opensans):
+        funcs = hb.DrawFuncs.create()
```

Comment:
we said `create()` classmethods are deprecated so it'd make sense to use default constructors in our own unit tests

Reply:
Why even add a deprecated method to a new object?

Reply:
Probably the honest answer is "because I don't *really* know what I'm doing with Cython so I'm just copy-and-pasting similar code and then hacking it until it works." Anyway, will fix.

Reply:
Wait, that is my mode of operation here, too :)